docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Transaction vs Continuous Profiling
+sidebar_order: 140
+description: "Continuous and transaction based profiling are two different modes with which you can profile the runtime of your application. This document serves as an explanation as to why we did it and what it enables."
+---
+
+<Note>
+We have released a new profiling mode called continuous profiling mode with fundamental differences from the previous transaction based profiling mode. This document serves as an explanation as to why we did so along with a couple of things that are helpful to know.
+</Note>
```

Comment:
```suggestion
We've released a new profiling mode called **continuous profiling**. Read on to learn about the differences between transaction-based and continuous profiling mode. 

```
I don't think this needs to be a note. It can just be the intro.